server/ansible/Inventory/pbench-server.hosts.example

```diff
@@ -1,9 +1,23 @@
+# Change the [servers] section to the set of hosts where you want to
+# install pbench-server. The various hosts can be running any
+# combinations of RHEL7, RHEL8, or (supported) Fedora versions
+# (currently 28 and 29).  They will also have different configuration
+# files which can be specifified using the value of the cenv variable
+# (which is used to set the value of the
+# pbench_configuration_environment variable which is use to retrieve
+# the configuration file from the URL in configurl) - see below for an
+# example.
+
 [servers]
-host1
-host1sat
+host1            cenv=pbench-master
+host1sat         cenv=pbench-satellite
+
+# N.B. pbench_install_dir must be set to the same value as in the
+# pbench-server.spec file that is used to build the pbench server RPM.
 
 [servers:vars]
-configurl = http://pbench.example.com/server/config/pbench-server.cfg
-configdest = /opt/pbench-server/lib/config
+pbench_install_dir = /opt/pbench-server
```

Comment:
Can we make `pbench_install_dir` a variable that has a default value and remove it from this example?  This would allow the example to be as minimal as possible.

We can still document that variable for use by the user in the README.

Reply:
Done. The variable is now named `pbench_server_install_dir'.